tests/components/miele/conftest.py

```diff
@@ -181,3 +182,10 @@ async def push_data_and_actions(
     action_callback = get_actions_callback(mock_miele_client)
     await action_callback(act_file)
     await hass.async_block_till_done()
+
+
+@pytest.fixture
+def mock_date() -> Generator[MagicMock]:
+    """Fixture to mock homeassistant.util.dt.utcnow()."""
+    with freeze_time("2025-05-31 12:30:00+00:00"):
+        yield
```

Comment:
Instead of building your own fixture, I think you can update the tests using it like this:
```py
@pytest.mark.freeze_time("2025-05-31 12:30:00+00:00")
```